blogs/artificial-intelligence/mamba2-flash-attention-kernels/README.md

```diff
@@ -0,0 +1,113 @@
+---
+blogpost: true
+date: 07 December 2024
+author: Quentin Anthony, Beren Millidge
+tags: LLM, AI/ML, GenAI, PyTorch
+category: Applications & models
+language: English
+myst:
+  html_meta:
+    "description lang=en": "Mamba2 and Flash Attention Backward Kernels on AMD MI300x with ROCm"
+    "keywords": "Mamba, PyTorch, S4, S6, Mamba2, Transformer, Flash Attention, Optimization, Hardware-aware, Transformer, Attention, ROCm, Mi210, MI250, MI300, AI/ML, Generative AI"
+    "property=og:locale": "en_US"
+---
+
+
+# Training Transformers and Hybrid models on AMD Instinct MI300X Accelerators
```

Comment:
Let me know what you think of this suggested tentative title (see metadata below).
Other suggestions are:
**Diamond in the Rough: Zyphra Speeding Up Training on AMD Instinct MI300X Accelerators**
Or simply:
**Zyphra Speeding Up Training on AMD Instinct MI300X Accelerators**
```suggestion
# Unleashing the Power: Zyphra Speeding Up Training on AMD Instinct MI300X Accelerators
```

Reply:
Title shared above "Zyphra Introduces Frontier Training Kernels for Transformers and SSMs on AMD Instinct MI300X Accelerators"